### PR TITLE
Issue and pr templates

### LINKED ISSUE #10: Create a PR Template and an Issue Template to make contributing easier
We need to create a pull request template and issue templates for this repo to make contributing to it easier.

You may reference the templates at the main [render engine](https://github.com/render-engine/render-engine) repo to help you.

We only need to create templates here just like the ones on the main repo.

### PR DESCRIPTION
**Pull request by JohananOppongAmoateng**
Added the issue and pr templates. 
Close #10

### CODE REVIEW

**Review by john0isaac — APPROVED** [review]
Thank you for contributing! I will merge this PR once this #15 pr is merged.

**Comment by john0isaac** [comment]
@JohananOppongAmoateng can you sync your branch with the latest changes that we just merged on the main branch? so the workflow passes?

**Comment by JohananOppongAmoateng** [comment]
Done

File: .github/PULL_REQUEST_TEMPLATE.md
Comment (john0isaac) [inline .github/PULL_REQUEST_TEMPLATE.md:4]:
```suggestion
## Checklist
```

**Pull request merged**
